forwards_paper.tex

```diff
@@ -1043,33 +1046,81 @@ \section*{Discussion}
 %%%%%%%%%%%%%%
 \section*{Methods}
 
-\krt{Are we doing a methods section?}
+\subsection*{The \cpp{} implementation}
 
-\subsection{Requirements for the \cpp{} implementation}
+\paragraph{Requirements.}
+The \cpp{} implementation uses several packages installed via conda,
+except for \msprime{}, which must be installed from GitHub.
 
 \begin{enumerate}
     \item \fwdpy{} 0.13.a0
     \item \fwdpp{} 0.5.7
     \item GNU Scientific Library 1.16
     \item pybind11 2.2.1
     \item GCC 4.8.5
-    \item \msprime{} master branch \jda{which commit hash or short version number
-		    from pip freeze or {\tt msprime.\_\_version\_\_}?}
+    \item \msprime{} 0.5.0 \jda{Changed these but the actual sims were not run
+		    with this version...}
 \end{enumerate}
 
-All of the above installed via conda, except for msprime, which must be installed from GitHub.
-
-\subsection{Requirements for the \simupop{} implementation}
-
-The simulations are implemented the Python package \ftprime{} 0.0.6rc0
-which is available at \url{https://www.github.com/ashander/ftprime}.
+\subsection*{The \simupop{} implementation}
+
+We implemented simulations in a Python script, usable as a command-line utility,
+taking input parameters for population size $N$ and scaled recombination $\rho$.
+As described in 'Simulation benchmarks', we set scaled mutation $\theta=\rho$,
+we used simplify interval of 1000 and simulated for $T=10N$ generations in each
+simulation, introducing deleterious mutations at a rate of $\rho/100$ per
+generation.
+Because \simupop{} requires explicit locations for loci, we set the number of
+loci $n=\rho / (4Nr)$ and marked
+$k={\rm round}(\mu / (100 (\mu + \mu / 100))) = 0$ of these as selected loci.
+\jda{unfortunately probably what we did...}
+The selected loci were chosen as a random sample of all the loci.
+
+We coupled \simupop{} to \msprime{} using a Python class \texttt{RecombCollector} 1)
+tracks time, and 2) collects and parses recombination information provided by
+the \texttt{Recombinator} class of \simupop{}, adding them to a \tnodetable{} and
+\tedgetable{} upon which simplification is performed every 1000 years (using
```

Comment:
Can we phrase this in generations? Years is confusing, as it sounds a bit like the simulator time.

Reply:
Thanks. Good call it's a tick from working with overlapping generations.